app/src/main/java/com/adroitandroid/p2pchat/ChatAdapter.java

```diff
@@ -53,7 +55,7 @@ void addMessage(String message, String senderName) {
 
         private final RowMessagesBinding binding;
 
-        MessageVH(RowMessagesBinding binding) {
+        MessageVH(@NotNull RowMessagesBinding binding) {
```

Comment:
why not use the android @NonNull instead https://developer.android.com/reference/androidx/annotation/NonNull.html